### ISSUE TITLE
No schemes included in the Xcode project for Bach

### ISSUE THREAD

**Issue by AlexHarker** [bachfamily/bach#4]
The Xcode project contains no schemes, presumably because these are under a user space, rather than shared, and/or your .gitignore is removing them. It may be that auto creating them is fine, but you can set schemes to be shared and thus they'll be included in the .git.

This is the approach I use in other projects where we are using Xcode projects - happy to answer any questions if this seems like a useful addition.

**Comment by danieleghisi** [comment]
Yes – there is an issue on that side. For the time being, you can safely go with "Autocreate schemes" though, and everything will be ok. Thanks for notifying.
